internal/distributed/proxy/httpserver/handler_v1.go

```diff
@@ -0,0 +1,535 @@
+		return
+	}
+	if httpReq.CollectionName == "" {
+		c.AbortWithStatusJSON(http.StatusOK, gin.H{HTTPReturnCode: http.StatusBadRequest, HTTPReturnMessage: "collectionName is required."})
+		return
+	}
+	has, err := h.hasCollection(c, httpReq.DbName, httpReq.CollectionName)
+	if err != nil {
+		return
+	}
+	if !has {
+		c.AbortWithStatusJSON(http.StatusOK, gin.H{HTTPReturnCode: http.StatusOK, HTTPReturnMessage: "can't find collection: " + httpReq.CollectionName})
+		return
+	}
+	req := milvuspb.DropCollectionRequest{
+		CollectionName: httpReq.CollectionName,
+	}
+	username, _ := c.Get(ContextUsername)
+	_, authErr := proxy.PrivilegeInterceptorWithUsername(c, username.(string), &req)
+	if authErr != nil {
+		c.JSON(http.StatusOK, gin.H{HTTPReturnCode: http.StatusUnauthorized, HTTPReturnMessage: authErr.Error()})
+		return
+	}
+	response, err := h.proxy.DropCollection(c, &req)
+	if err != nil {
+		c.JSON(http.StatusOK, gin.H{HTTPReturnCode: http.StatusBadRequest, HTTPReturnMessage: "drop collection " + httpReq.CollectionName + " fail", HTTPReturnError: err.Error()})
+	} else if response.ErrorCode != commonpb.ErrorCode_Success {
+		c.JSON(http.StatusOK, gin.H{HTTPReturnCode: response.ErrorCode, HTTPReturnMessage: "drop collection " + httpReq.CollectionName + " fail", HTTPReturnError: response.Reason})
+	} else {
+		c.JSON(http.StatusOK, gin.H{HTTPReturnCode: http.StatusOK, HTTPReturnData: gin.H{}})
+	}
+}
+
+func (h *Handlers) query(c *gin.Context) {
+	httpReq := QueryReq{
+		DbName: DefaultDbName,
+		Limit:  100,
+	}
+	if err := c.ShouldBindBodyWith(&httpReq, binding.JSON); err != nil {
+		c.AbortWithStatusJSON(http.StatusOK, gin.H{HTTPReturnCode: http.StatusBadRequest, HTTPReturnMessage: "check your parameters conform to the json format", HTTPReturnError: err.Error()})
+		return
+	}
+	if httpReq.CollectionName == "" {
+		c.AbortWithStatusJSON(http.StatusOK, gin.H{HTTPReturnCode: http.StatusBadRequest, HTTPReturnMessage: "collectionName is required."})
+		return
+	}
+	req := milvuspb.QueryRequest{
+		CollectionName:     httpReq.CollectionName,
+		Expr:               httpReq.Filter,
+		OutputFields:       httpReq.OutputFields,
+		GuaranteeTimestamp: BoundedTimestamp,
+	}
+	username, _ := c.Get(ContextUsername)
+	_, authErr := proxy.PrivilegeInterceptorWithUsername(c, username.(string), &req)
+	if authErr != nil {
+		c.JSON(http.StatusOK, gin.H{HTTPReturnCode: http.StatusUnauthorized, HTTPReturnMessage: authErr.Error()})
+		return
+	}
+	response, err := h.proxy.Query(c, &req)
+	if err != nil {
+		c.JSON(http.StatusOK, gin.H{HTTPReturnCode: http.StatusBadRequest, HTTPReturnMessage: "query fail", HTTPReturnError: err.Error()})
+	} else if response.Status.ErrorCode != commonpb.ErrorCode_Success {
+		c.JSON(http.StatusOK, gin.H{HTTPReturnCode: response.Status.ErrorCode, HTTPReturnMessage: response.Status.Reason})
+	} else {
+		outputData, err := buildQueryResp(int64(0), response.OutputFields, response.FieldsData, nil)
```

Comment:
It is ill-advised to directly utilize response.OutputFields in this instance, as the OutputFields here may potentially encompass the names of fields within the schema. Hence, it is imperative to exclude those originating from the schema.
Here is the example code:
```
nonDynamicFieldNames := make(map[string]struct{})
for _, field := range response.FieldsData {
    if !field.IsDynamic {
        nonDynamicFieldNames[field.Name] = struct{}{}
    }
}

dynamicFields := []string{}

for _, fieldName := range response.OutputFields {
    if _, exist := nonDynamicFieldNames[fieldName]; !exist {
        dynamicFields = append(dynamicFields, fieldName)
    }
}
```